test/module/irohad/ametsuchi/block_query_test.cpp

```diff
@@ -212,3 +215,154 @@ TEST_F(BlockQueryTest, GetTransactionsWithInvalidTxAndValidTx) {
   });
   ASSERT_TRUE(wrapper.validate());
 }
+
+/**
+ * @given block store with 2 blocks totally containing 3 txs created by
+ * user1@test AND 1 tx created by user2@test
+ * @when get non-existent 1000 block
+ * @then nothing is returned
+ */
+TEST_F(BlockQueryTest, BlockQuery_GetBlocks_GetNonExistentBlock) {
+  auto wrapper = make_test_subscriber<CallExact>(
+      blocks->getBlocks(1000, 1), 0);
```

Comment:
Seem there's missing edge-case test (e.g getting 2nd blocks)

Reply:
1. there are blocks 1, 2
2. test name is "Get Non Existent Block". 1000 is more safe option.